template-parts/global-templates/hero-pagetitle.php

```diff
@@ -5,14 +5,41 @@
 	$title = cpschool_get_page_title();
 	?>
 	<header id="hero-main" <?php cpschool_class( 'hero-main', 'hero jumbotron jumbotron-fluid has-background has-hero-main-bg-color-background-color' ); ?> aria-label="<?php esc_html_e( 'page title and basic information', 'cpschool' ); ?>">
+		<?php
```

Comment:
I wonder if it would not be better to create a function out of it since almost identical code is being used below. Also, on front-end (not customizer) it would be better to render it only when its actually being used.

Reply:
So what about turning it into the function?